src/components/MusicView.tsx

```diff
@@ -42,7 +42,7 @@ const MusicView: React.FC<MusicViewProps> = (props) => {
     <div>
       {/* Second Tune Toggler  */}
       {songHasTwoTunes && (
-        <div id="songTogglerDiv">
+        <div id="songToggler">
```

Comment:
these ids are tied to css styles, can you update the `Components.css` file and change the `#songTogglerDiv` to `#songToggler` and also for `musicView`